### ISSUE TITLE
scrolledUp doesn't seem to work properly

### ISSUE THREAD

**Issue by murphmar** [orizens/ngx-infinite-scroll#250]
## Expected Behavior
Scrolling to the top of a container with the default settings should trigger the scrolledUp binding

## Actual Behavior
Nothing happens. If the defaults are changed to numbers that don't really make sense in execution, ie: scrolledUpDistance="9", the call is triggered. 

### Steps To Reproduce / Demo (if applicable)
Here's a simple stackblitz recreating the issue: https://stackblitz.com/edit/angular-9nx2jt?file=app%2Fapp.component.html

### Context
Was intending to use in a much more involved situation, but can't get it working even in a stripped down scenario.

### Your Environment
Angular 5.2.8, matching the stackblitz

I very well could be missing something simple here.

**Comment by asterius321** [comment]
Same problem for me

**Comment by AndrewEckart** [comment]
Had the same problem and drove myself crazy trying to figure out why before I realized that it was introduced when I upgraded my dependency from 0.5.2 to 0.8.3 a while back. I downgraded for now - no issues running 0.5.2 with Angular 6, and I still need `rxjs-compat` for a few other dependencies anyway. The Stackblitz repro above also uses 0.8.3, so I suspect it's a bug with the library introduced in one of the recent releases.
